src/resources/editor/tools/yaml/yaml-intelligence-resources.json

```diff
@@ -4096,6 +4113,26 @@
           }
         }
       }
+    },
```

Comment:
@dragonstyle I suspect this is because you (or someone) updated the YML without running `build-js` so it is coming along for a ride on my PR.  Is this a problem or should I just merge it in?